packages/node/__tests__/integration.spec.ts

```diff
@@ -81,8 +81,11 @@ describe('Masked Error Option', () => {
   it('should mask error', async () => {
     const yoga = createServer({
       schema,
-      maskedErrors: true,
       logging: false,
+      plugins: getDefaultPlugins({
```

Comment:
So the usage now is that I have to manually invoke what feels like an internal helper from Yoga in order to opt in to Yoga's default plugins?